winui/DataGrid/Autosize-Columns.md

```diff
@@ -123,6 +123,44 @@ this.sfDataGrid.Columns["OrderID"].ColumnWidthMode = ColumnWidthMode.AutoLastCol
 
 <img src="Autosize-Columns-images/winui-datagrid-column-filling.png" alt="Specific Column in WinUI DataGrid Fills Remaining Width" width="100%" Height="Auto"/>
 
+## Improving Performance of Auto-Sizing Calculations
+
+By default, the column auto-sizing calculation was performed for all rows in the DataGrid. However, we now provide support for the `AutoFitRange` property, which allows auto-sizing to be calculated for either visible rows or all rows in the grid. The default value is `AutoFitRange.AllRows`.
+While scrolling, the column width adjusts according to the visible text. When you scroll up, the width remains unchanged and retains its maximum value. Additionally, when a new record is added or edited, the column width adjusts based on the text content.
```

Comment:
Width remains unchanged ? content is improper.

Reply:
@sethuramkumar  - Okay , I have modified the content.